### PR TITLE
Sort term lists

### PR DESCRIPTION
**Pull request by cmurtaugh**
This PR adds a new JS file that will help make the Canvas UI more usable by admins by sorting terms by their labels. The term dropdown list on the account courses page and the table on the account terms page will be sorted in a way that should make it easier to find specific entries. 

This custom JS is currently installed in our test instance of Canvas.

### CODE REVIEW

File: js/sort_term_dropdowns.js [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+  var middleList = new Array();
+  var bottomList = new Array();
+
+  for (var i=0; i < termSelectList.length; i++) {
+    if (termSelectList[i].label == 'All Terms') {
+      // this should be the first item in the list
+      topList.unshift(termSelectList[i]);
+    }
+    else if (termSelectList[i].label == 'Ongoing') {
+      // this should be the second item in the list
```
Comment (sapnamysore) [inline js/sort_term_dropdowns.js:19]: Minor: Would it be safe to have a case insensitive match, although looks like Canvas has been using 'Ongoing' consistently.

File: js/sort_term_dropdowns.js [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+    }
+    else {
+      // term labels that start with a letter are at the bottom, sorted ascending
+      bottomList.push(termRowList[i]);
+    }
+  }
+
+  var newList = topList.sort(function(a,b){
+      var alabel = a.getElementsByClassName('name')[0].innerHTML;
+      var blabel = b.getElementsByClassName('name')[0].innerHTML;
```
Comment (sapnamysore) [inline js/sort_term_dropdowns.js:63]: Nitpick: camelCase for the vars above?

**Comment by sapnamysore** [comment]
👍

File: js/sort_term_dropdowns.js [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+      return (alabel >= blabel) ? -1 : 1;
+    }).concat(
+      bottomList.sort(function(a,b){
+        var alabel = a.getElementsByClassName('name')[0].innerHTML;
+        var blabel = b.getElementsByClassName('name')[0].innerHTML;
+        return (alabel >= blabel) ? 1 : -1;
+      })
+    );
+
+  $('table#terms').children('tbody').append(newList);
```
Comment (elliottyates) [inline js/sort_term_dropdowns.js:74]: I'm interested to know how this line works, when you have a chance. It's unintuitive to me. Thanks!

**Pull request merged**